test/burials/config/documents.unit.spec.jsx

```diff
@@ -0,0 +1,133 @@
+import React from 'react';
+import { expect } from 'chai';
+import sinon from 'sinon';
+import ReactTestUtils from 'react-dom/test-utils';
+
+import { DefinitionTester, getFormDOM } from '../../util/schemaform-utils.jsx';
+import formConfig from '../../../src/js/burials/config/form';
+
+describe('Burials document upload', () => {
+  const { schema, uiSchema, depends } = formConfig.chapters.additionalInformation.pages.documentUpload;
+
+  it('should render', () => {
+    const form = ReactTestUtils.renderIntoDocument(
+      <DefinitionTester
+          schema={schema}
+          definitions={formConfig.defaultDefinitions}
+          uiSchema={uiSchema}/>
+    );
+    const formDOM = getFormDOM(form);
+
+    expect(formDOM.querySelectorAll('input,select').length).to.equal(0);
+  });
+
+  it('should render death certificate field', () => {
+    const form = ReactTestUtils.renderIntoDocument(
+      <DefinitionTester
+          schema={schema}
+          data={{
+            burialAllowanceRequested: 'service'
+          }}
+          definitions={formConfig.defaultDefinitions}
+          uiSchema={uiSchema}/>
+    );
+    const formDOM = getFormDOM(form);
+
+    expect(formDOM.querySelectorAll('input,select').length).to.equal(1);
+  });
+
+  it('should render receipts field', () => {
+    const form = ReactTestUtils.renderIntoDocument(
+      <DefinitionTester
+          schema={schema}
+          data={{
+            'view:claimedBenefits': {
+              transportation: true
+            }
+          }}
+          definitions={formConfig.defaultDefinitions}
+          uiSchema={uiSchema}/>
+    );
+    const formDOM = getFormDOM(form);
+
+    expect(formDOM.querySelectorAll('input,select').length).to.equal(1);
+  });
+
```

Comment:
Would it provide value to have a test that makes sure they both render together? I mean, I see that the next test does render them both and checks that they both have errors, so maybe that's sufficient..?